sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/Models/TelemetryItem.cs

```diff
@@ -25,18 +25,29 @@ internal partial class TelemetryItem
 
             if (activity.GetTelemetryType() == TelemetryType.Request)
             {
-                if (activityTagsProcessor.activityType.HasFlag(OperationType.V2))
+                // All of this block should be changed to:
+                // Tags[ContextTagKeys.AiOperationName.ToString()] = activity.DisplayName.Truncate(SchemaConstants.Tags_AiOperationName_MaxLength);
+                // 
```

Comment:
All of this block should be changed to this one line, but I'm leaving the long version in for review and discussion.